packages/search-types/src/query-signals/preview-results.model.ts

```diff
@@ -0,0 +1,13 @@
+import { Result } from '../result/result.model';
+
+/**
+ * Interface to type the query preview objects.
+ */
+export interface PreviewResults {
```

Comment:
Move this file outside the `query-signals` folder, as it is no longer a next-queries dependant feature